image-scripts/include/chimera.sh

```diff
@@ -0,0 +1,56 @@
+require_cmd wget
+
+BASEURL=https://repo.chimera-linux.org/live/latest
+ROOTFS=
+
+fetch() {
+	local name
+	local rx
+
+	rx='chimera-linux-x86_64-ROOTFS-\d+-bootstrap.tar.gz'
+
+	# Fetch checksums to find out latest release name
+	wget -O - "$BASEURL/sha256sums.txt" | grep -P "$rx" > "$DOWNLOAD/sha256sums.txt"
+
+	# Extract the name
+	name=$(grep -o -P "$rx" "$DOWNLOAD/sha256sums.txt")
+
+	# Download rootfs
+	wget -P "$DOWNLOAD" "$BASEURL/$name"
```

Comment:
No verification via `minisig`?

Reply:
the other image scripts don't, and i'm not sure if minisig is present in the environment/if i can get it there somehow